test/e2e_kubeadm/nodes_test.go

```diff
@@ -58,6 +58,8 @@ var _ = Describe("nodes", func() {
 			fgCC := cc["featureGates"].(map[interface{}]interface{})
 			if fg, ok := fgCC["NodeLocalCRISocket"]; ok {
 				nodeLocalCRISocketEnabled = fg.(bool)
+			} else {
+				nodeLocalCRISocketEnabled = true
```

Comment:
i guess this is not a valid fix because it assumes the featureGates key exists.

try

```
		nodeLocalCRISocketEnabled := true
		cc := getClusterConfiguration(f.ClientSet)
		if _, ok := cc["featureGates"]; ok {
			fgCC := cc["featureGates"].(map[interface{}]interface{})
			if fg, ok := fgCC["NodeLocalCRISocket"]; ok {
				nodeLocalCRISocketEnabled = fg.(bool)
...
```